providers/sftp/pyproject.toml

```diff
@@ -59,7 +59,7 @@ requires-python = ">=3.10"
 dependencies = [
     "apache-airflow>=2.10.0",
     "apache-airflow-providers-ssh>=4.0.0",
-    "paramiko>=2.9.0",
+    "paramiko>=2.9.0,<4.0.0",
```

Comment:
lets have todo + link to github issue that explain what needs to be fixed to be compatible with 4.0

Reply:
Thanks, added it in the commit: https://github.com/apache/airflow/pull/54078/commits/16b4b64d4e6b234dc5e2c2732d7f98a08fb0bc2c